projects/plugins/jetpack/_inc/client/components/section-nav/style.scss

```diff
@@ -345,6 +345,23 @@
 	}
 }
 
+// Mimic style of WP plugin update badge
+.dops-section-nav-tab__update-badge {
+	display: inline-block;
+	vertical-align: top;
+	box-sizing: border-box;
+	margin: 1px 0 -1px 2px;
+	padding: 0 5px;
+	min-width: 18px;
+	height: 18px;
+	border-radius: 9px;
+	background-color: #d63638;
```

Comment:
Maybe you'd be able to pull this from https://github.com/Automattic/jetpack/blob/3238c5d6df6727ce64fc18763a0025f10920a317/projects/plugins/jetpack/_inc/client/scss/variables/_colors.scss#L48

Reply:
Ahh, nice. Thanks for the reference 👍